testutil/tx/eth.go

```diff
@@ -59,7 +61,13 @@ func PrepareEthTx(
 		msg.From = ""
 
 		txGasLimit += msg.GetGas()
-		txFee = txFee.Add(sdk.Coin{Denom: utils.BaseDenom, Amount: sdkmath.NewIntFromBigInt(msg.GetFee())})
+		// NOTE: This is not always going be true, but we assume 6 decimals
+		// for the Evmos chain is always going to be uatom
+		if evmParams.DenomDecimals == types.Denom6Dec {
+			txFee = txFee.Add(sdk.Coin{Denom: "uatom", Amount: sdkmath.NewIntFromBigInt(msg.GetFee())})
+		} else {
+			txFee = txFee.Add(sdk.Coin{Denom: utils.BaseDenom, Amount: sdkmath.NewIntFromBigInt(msg.GetFee())})
```

Comment:
Use the evm denom from the evm params on the fees
Also, need to scale the amount to 6 decimals on the 
`evmParams.DenomDecimals == types.Denom6Dec` case
```suggestion
		if evmParams.DenomDecimals == types.Denom6Dec {
			txFee = txFee.Add(sdk.Coin{Denom: evmParams.EvmDenom, Amount: sdkmath.NewIntFromBigInt(msg.GetFee())})
		} else {
			txFee = txFee.Add(sdk.Coin{Denom: evmParams.EvmDenom, Amount: sdkmath.NewIntFromBigInt(msg.GetFee())})
```